app/src/main/kotlin/crazyboyfeng/accSettings/acc/AccHandler.kt

```diff
@@ -8,10 +8,11 @@ import java.io.File
 import java.io.FileOutputStream
 import java.io.IOException
 
-
 class AccHandler {
-    private suspend fun install(context: Context) {
-        suspend fun cacheAssertFile(fileName: String): File = withContext(Dispatchers.IO) {
```

Comment:
This is not a spelling mistake, `asset` means resource, and here it points to Android's `asset` resource directory. `assert` is a kind of declaration.